compiler/rustc_mir/src/transform/const_goto.rs

```diff
@@ -0,0 +1,122 @@
+//! This pass optimizes the following sequence
+//! ```rust
```

Comment:
This provably needs `,ignore` appended to prevent rustdoc from trying to run it when testing.

Reply:
Thanks, fixed.